code/modules/atmospherics/machinery/components/tank.dm

```diff
@@ -59,7 +59,9 @@
 
 /obj/machinery/atmospherics/components/tank/Initialize()
 	. = ..()
-
+	for (var/datum/gas_mixture/airs_ports as anything in airs)
+		airs_ports.volume = 0 // so that all gas shows up on the analyzer properly, these nodes aren't actually needed for functionality here
+	
 	if(!knob_overlays)
```

Comment:
This feels really hacky, and I'm uncomfortable with the fact that it's needed. Are these mixes totally unused? why do they need to be set to 0 volume in the first place?